### PR TITLE
Add expected Masterful Blitz count

### PR DESCRIPTION
**Pull request by skimmedsquare**
Addresses part of #1303

P1S:
Screen Shot 2022-05-03 at 3 08 09 PM

P3S: 
Screen Shot 2022-05-03 at 3 08 59 PM

## Update (5/4):
Added suggestion based on blitz count
image

### CODE REVIEW

File: src/parser/jobs/mnk/modules/evaluators/BlitzEvaluator.tsx [hunk 1]
```diff
@@ -1,41 +1,134 @@
...
+		return new TieredSuggestion({
+			icon: this.blitzIcon,
+			content: <Trans id="mnk.rof.suggestions.blitz.content">
+				Try to hit two uses of <DataLink action="MASTERFUL_BLITZ"/> in both the opener and every 'even' <DataLink action="RIDDLE_OF_FIRE"/> window and one usage of <DataLink action="MASTERFUL_BLITZ"/> in every 'odd' window, as the blitz actions are your strongest skills.
+			</Trans>,
+			why: <Trans id="mnk.rof.suggestions.blitz.why">
+				<Plural value={missedBlitzes} one="# use of" other="# uses of"/> <DataLink action="MASTERFUL_BLITZ"/> <Plural value={missedBlitzes} one="was" other="were"/> missed during {badWindows.length} <DataLink action="RIDDLE_OF_FIRE"/> <Plural value={badWindows.length} one ="window" other="windows"/>
+			</Trans>,
+			tiers: BLITZ_SEVERITY_TIERS,
+			value: missedBlitzes,
```
Comment (ghost) [inline src/parser/jobs/mnk/modules/evaluators/BlitzEvaluator.tsx]: Same nit as the Brotherhood module
```suggestion
			tiers: BLITZ_SEVERITY_TIERS,
			value: missedBlitzes,
			why: <Trans id="mnk.rof.suggestions.blitz.why">
				<Plural value={missedBlitzes} one="# use of" other="# uses of"/> <DataLink action="MASTERFUL_BLITZ"/> <Plural value={missedBlitzes} one="was" other="were"/> missed during {badWindows.length} <DataLink action="RIDDLE_OF_FIRE"/> <Plural value={badWindows.length} one ="window" other="windows"/>
			</Trans>,
```

**Review by ghost — COMMENTED** [review]
Just one small nit on this one.

**Review by ghost — APPROVED** [review]
LGTM. Still needs a review by Acc or Ack.

**Review by AkairyuGestalter — APPROVED** [review]
Changes seem to be working as expected as far as I can tell. Can continue to polish on future PRs if we find issues later on.

**Pull request merged**